CODEOWNERS

```diff
@@ -26,10 +26,14 @@ src/audio/tone.c			@singalsu
 src/arch/xtensa/*			@tlauda
 src/platform/*				@tlauda
 src/platform/baytrail/*			@xiulipan
-src/platform/haswell/*			@xiulipan
+src/platform/haswell/*			@xiulipan @randerwang
 
 # drivers
 src/drivers/intel/cavs/dmic.c		@singalsu
+src/drivers/intel/cavs/hda.c		@randerwang
```

Comment:
I think you don't want to do this. It makes you one and only codeowner of hda.
It's not just a watchlist. You can watch only on repo level by using github built-in features (watch on the top).
If you want to get notifications on directory/file-level you can try something like in this answer: https://stackoverflow.com/a/25315476

Reply:
sorry, I don't make it clear in my commit message. I want to review change in some files.
I will refine my file list.